### PR TITLE
added penthouse timeout option

### PR DESCRIPTION
**Pull request by leogdion**
Often times Penthouse times out. I allowed the ability to set the timeout used by Penthouse.

### CODE REVIEW

File: .nvmrc [hunk 1]
```diff
@@ -0,0 +1 @@
+stable
```
Comment (addyosmani) [inline .nvmrc]: Can we drop the nvmrc?

**Comment by addyosmani** [comment]
One nit. Thanks

File: .nvmrc (hunk 1 above)
Comment (leogdion) [inline .nvmrc]: Yes

File: README.md [hunk 2]
```diff
@@ -219,6 +222,7 @@ critical.generate({
 | inlineImages     | `boolean`     | `false` | Inline images
 | assetPaths       | `array`       | `[]` | List of directories/urls where the inliner should start looking for assets
 | maxImageFileSize | `integer`     | `10240`| Sets a max file size (in bytes) for base64 inlined images
+| timeout | `integer`     | `30000`| Sets a maximum timeout for the operation
```
Comment (sindresorhus) [inline README.md]: The column separators should be aligned with the others.

Comment (addyosmani) [inline README.md]: friendly ping on this @leogdion

Comment (leogdion) [inline README.md]: done. let me know if there is anything else.

**Pull request merged**